gunicorn/arbiter.py

```diff
@@ -152,7 +153,8 @@ def start(self):
                 for fd in os.environ.pop('GUNICORN_FD').split(','):
                     fds.append(int(fd))
 
-            self.LISTENERS = sock.create_sockets(self.cfg, self.log, fds)
```

Comment:
unsuer about that change. Why forcing the port reusing?

Reply:
if we want to use reuse port (and the underlying platform supports it), then we want to make the sockets in the worker process (line 584).  If not, we stick to the existing path

Reply:
@benoitc ping ^

Reply:
this should only be an option. Even if the sockets can enable this i don't want to force this for now  The detach part may be OK. I still have to test this change. I was focusing on some other internal change. I will test it tomorrow.

Reply:
Sorry, I misread the condition `if not (self.cfg.reuse_port and hasattr(socket, 'SO_REUSEPORT')):`

i think i would be better to move the capability testing in the config module to be more explicit and just do `if not self.cfg.reuse_port` here. That would also give the opportunity to raise a warning if it is set to true but the system is not capable of i t. Otherwis I think the patch is probably ok. I need to test it.